src/hotspot/share/runtime/deoptimization.cpp

```diff
@@ -1834,6 +1838,26 @@ static void post_deoptimization_event(CompiledMethod* nm,
 
 #endif // INCLUDE_JFR
 
+void log_deopt(CompiledMethod* nm, Method* tm, intptr_t pc, frame& fr, int trap_bci,
```

Comment:
Add 'static' keyword so it has internal linkage.

Reply:
Fixed.